3rdParty/fuerte/include/fuerte/message.h

```diff
@@ -270,7 +272,14 @@ class Response final : public Message {
   void setPayload(velocypack::Buffer<uint8_t> buffer,
                   std::size_t payloadOffset);
 
+  void addSupportedAuth(std::string const& value);
+  bool isMethodAllowed(AuthenticationType meth) {
+    return std::find(_supportedAuths.begin(),
+                     _supportedAuths.end(),
+                     meth) != _supportedAuths.end();
+  }
  private:
+  std::vector<AuthenticationType> _supportedAuths;
```

Comment:
no wtf

Reply:
just use an enum if it has to be done

Reply:
AuthenticationType is an enum? Rather a bitfield?